### ISSUE TITLE
[BUG] Elvis operator not working correctly (`?:`)

### ISSUE THREAD

**Issue by ModProg** [elkowar/eww#623]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

Elvis operator works on empty strings but not on null, i.e. `"null"?:"hi"` will result in `null` but `""?:"hi"` will result in `hi`.

I also tested it with creating null by indexing an array (`[][1]`), to make sure it is not just that the string `"null"` is handled differently.

### Reproducing the issue

```lisp
(defwindow test
  :monitor 0
  (label :text {"[]"[1]?: "hi"}))
```

### Expected behaviour

Should show `hi` but shows `null`

### Additional context

I noticed that while implementing #622 in yuck.

**Comment by viandoxdev** [comment]
null and the string null are treated the exact same way by simplexpr IIRC, there is no json value only json operators

**Comment by ModProg** [comment]
> null and the string null are treated the exact same way by simplexpr IIRC, there is no json value only json operators

Thought so too, just wanted to make sure.

**Comment by ModProg** [comment]
This was fixed since 0.4.0
